internal/testrunner/runners/system/tester.go

```diff
@@ -1870,14 +1890,17 @@ func createPackageDatastream(
 	kibanaPolicy kibana.Policy,
 	pkg packages.PackageManifest,
 	policyTemplate packages.PolicyTemplate,
-	ds packages.DataStreamManifest,
+	ds *packages.DataStreamManifest,
 	config testConfig,
 	suffix string,
-) kibana.PackageDataStream {
+) (kibana.PackageDataStream, error) {
 	if pkg.Type == "input" {
-		return createInputPackageDatastream(kibanaPolicy, pkg, policyTemplate, config, suffix)
+		return createInputPackageDatastream(kibanaPolicy, pkg, policyTemplate, config, suffix), nil
+	}
+	if ds == nil {
+		return kibana.PackageDataStream{}, fmt.Errorf("data stream manifest is required for integration packages")
```

Comment:
This error assumes that by not being an input package this is an integration package.
This is fine by now, but may not be in the future. In any case we will need to review this code if we introduce system tests in other types of packages.